packages/react-router-permissions/src/permissions-provider/permissions-provider.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import React from 'react';
+import { PermissionsContext } from '../permissions-context/permissions.context';
+
+export type PermissionsProviderProps = {
+  permissions: *,
+  authorizationStrategy: (*, *) => *,
```

Comment:
To make this generic type in a relevant way - so that types are kept throughout consumer provider and context, we would need to export factory function instead of components and we decided we do not want that.